### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by DurgNomis-drol**
Just a little bit of cleanup, so we have a nice base to work from.

### CODE REVIEW

File: custom_components/ha-glocaltokens/api.py [hunk 1]
```diff
@@ -54,37 +53,42 @@ def get_android_id(self):
         return self._client.get_android_id()
 
     def create_url(self, ip, port, api_endpoint):
+        """Creates url to endpoint.
+        Note: port argument is unused because all request must be done to 8443"""
```
Comment (leikoilja) [inline custom_components/ha-glocaltokens/api.py:57]: Great comment!

File: custom_components/ha-glocaltokens/api.py [hunk 2]
```diff
@@ -54,37 +53,42 @@ def get_android_id(self):
...
         )
         return url
 
     def get_alarms_and_timers_from(self, device, endpoint):
+        """Fetches timers and alarms from google device"""
         url = self.create_url(device.ip, device.port, endpoint)
         _LOGGER.debug(
             "For device {device} - {url}".format(device=device.device_name, url=url)
         )
+        # verify=False is need to avoid SSL security checks. Othervise it will fail to connect"""
```
Comment (leikoilja) [inline custom_components/ha-glocaltokens/api.py:69]: I think we misspelled "othervise" 😄

File: custom_components/ha-glocaltokens/api.py [hunk 3]
```diff
@@ -54,37 +53,42 @@ def get_android_id(self):
...
+        response = requests.get(url, headers=HEADERS, verify=False, timeout=TIMEOUT)
 
         if response.status_code != HTTP_OK:
             _LOGGER.error(
-                "For device {device} - API returned {error}".format(
-                    device=device.device_name, error=response.status_code
+                "For device {device} - API returned {error}: {reason}".format(
+                    device=device.device_name,
+                    error=response.status_code,
+                    reason=response.reason,
```
Comment (leikoilja) [inline custom_components/ha-glocaltokens/api.py:78]: Are you sure the error message lands in `response.reason`? Some other http requests i have worked with were returning error message in `response.message`, but i m not sure what would be here.

File: custom_components/ha-glocaltokens/const.py [hunk 4]
```diff
@@ -31,18 +30,21 @@
...
-# API
+# DEVICE PORT
 PORT = 8443
+
+# API
 API_ENDPOINT_ALARMS = "setup/assistant/alarms"
 HEADER_CAST_LOCAL_AUTH = "cast-local-authorization-token"
 HEADER_CONTENT_TYPE = "content-type"
-
+# HEADERS
```
Comment (leikoilja) [inline custom_components/ha-glocaltokens/const.py:40]: Maybe just an extra empty line above would help being consistent?

File: custom_components/ha-glocaltokens/const.py [hunk 5]
```diff
@@ -55,17 +57,14 @@
...
 
 # Access token only lives about 1 hour
 # Update often to fetch timers in timely manner
 SCAN_INTERVAL = 15  # Seconds
 
-DEVICE_NAME = "deviceName"
-DEVICE_IP = "ip"
-DEVICE_PORT = "port"
-TOKEN = "localAuthToken"
+# JSON name when retrieving information fra devices
```
Comment (leikoilja) [inline custom_components/ha-glocaltokens/const.py:67]: Maybe something like "json parameter values when retrieving information from devices"?

**Review by leikoilja — APPROVED** [review]
Great cleanup, thank you! 💯 🚀 

I will approve, feel free to either fix/skip the comments I have added, but once fixed - let's merge it in 💥

File: custom_components/ha-glocaltokens/api.py (hunk 2 above)
Comment (DurgNomis-drol) [reply custom_components/ha-glocaltokens/api.py:70]: :facepalm:

File: custom_components/ha-glocaltokens/api.py (hunk 3 above)
Comment (DurgNomis-drol) [reply custom_components/ha-glocaltokens/api.py:79]: As far as i can see requests lib does not return message but instead reason if it fails [requests-lib](https://requests.readthedocs.io/en/master/api/#requests.Response)

We can also change it to `response.text` instead?

Comment (DurgNomis-drol) [reply custom_components/ha-glocaltokens/api.py]: I have changed it to `response.text` because it will return the most amount of info in case of error

**Comment by leikoilja** [comment]
Superb, thanks 🚀

**Pull request merged**